Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/DownloadPackageCommand.cs

```diff
@@ -87,14 +90,16 @@ public override void Execute(Cancellation.CancellationToken cancellationToken)
             var downloadStartTime = DateTime.Now;
 
             var stalledTimeout = TimeSpan.FromSeconds(10);
-            
+
             using (_status.BytesPerSecond.Subscribe(bps =>
-                _status.Description.Value = bps > 0.01 || DateTime.Now - downloadStartTime < stalledTimeout ? "Downloading package..." : "Stalled..."))
+                _status.Description.Value = bps > 0.01 || DateTime.Now - downloadStartTime < stalledTimeout
+                    ? PatcherLanguages.OpenTag + "downloading_package" + PatcherLanguages.CloseTag
+                    : PatcherLanguages.OpenTag + "stalled" + PatcherLanguages.CloseTag))
```

Comment:
Create LanguageHelper and use import static, e.g. `Tag("stalled")`

Reply:
We're using c # 4.0. There is no import static yet

Reply:
done